frontend/src/components/popup/accept_transfer_popup.ts

```diff
@@ -57,10 +57,14 @@ class TransferPopup extends MobxLitElement {
     this.analyticsService.trackButtonClick(ButtonClick.TRANSFER_ACCEPT);
     this.participantService.acceptParticipantTransfer();
 
-    // Move to next stage.
-    const nextStageId = this.experimentService.getNextStageId(
-      this.participantService.profile?.currentStageId!
-    )!;
+    // Move to next stage if on a transfer stage.
```

Comment:
@cjqian what do you think about moving this routing logic to acceptParticipantTransfer (called a few lines up where we already have "set current stage to next if transfer stage" logic? Happy to make the change in a new branch/PR, or if you're working on anything similar, feel free to take it

Reply:
Update: I can take this. It might be related to the transfer stage refresh bug I'm currently investigating.